packages/docs-examples/components/select/select-footer/select-footer-example.ts

```diff
@@ -12,8 +28,40 @@ import { KbqSelectModule } from '@koobiq/components/select';
     standalone: true,
     selector: 'select-footer-example',
     templateUrl: 'select-footer-example.html',
-    imports: [KbqFormFieldModule, KbqSelectModule, KbqButtonModule, KbqIconModule, KbqLinkModule]
+    imports: [KbqFormFieldModule, KbqSelectModule, KbqButtonModule, KbqIconModule, KbqLinkModule],
+    styles: `
+        .layout-row {
+            width: 400px;
+            margin: 0 auto;
+            padding: 16px;
+            align-items: center;
+            gap: 24px;
+            justify-content: flex-end;
+        }
+
+        .kbq-form__label {
+            white-space: nowrap;
+            color: var(--kbq-foreground-contrast-secondary);
+        }
+    `
 })
 export class SelectFooterExample {
-    selected = '';
+    selectedButton = '';
+    selectedCaption = '';
+    selectedLink = '';
+
+    options: string[] = [];
+
+    constructor(@Inject(KBQ_LOCALE_SERVICE) private localeService: KbqLocaleService) {
+        this.localeService.changes.subscribe(this.update);
+    }
+
+    update = (locale: string) => {
+        this.options = localeDataSet[locale].items;
+        this.selectedButton = localeDataSet[locale].items[0];
+        this.selectedCaption = localeDataSet[locale].items[0];
+        this.selectedLink = localeDataSet[locale].items[0];
+    };
+    protected readonly colors = KbqComponentColors;
```

Comment:
```suggestion
    readonly colors = KbqComponentColors;
```
кажется что в примере можно не париться с модификаторами, `readonly` - ок

переменные лучше определять в начале